devops/scripts/install_deps_ubuntu2004.sh

```diff
@@ -4,4 +4,7 @@ apt -y update
 apt-get -y install software-properties-common
 apt-add-repository ppa:lttng/stable-2.10 -y
 apt -y update
-apt-get -y install git cmake build-essential curl libcurl4-openssl-dev libssl-dev uuid-dev libgtest-dev libgmock-dev liblttng-ust-dev rapidjson-dev ninja-build wget
\ No newline at end of file
+apt-get -y install git cmake build-essential curl libcurl4-openssl-dev libssl-dev uuid-dev libgtest-dev libgmock-dev liblttng-ust-dev rapidjson-dev ninja-build wget
+# Install Dotnet - Needed for e2e tests
+# See https://docs.microsoft.com/en-us/dotnet/core/install/linux-ubuntu#1804-
+curl -sSL https://dot.net/v1/dotnet-install.sh | bash /dev/stdin
```

Comment:
Instructions for manually building OSConfig on Ubuntu 18.04 and 20.04 are identical. Why are these scripts here different? Can we have a single script for all Ubuntu targets?

Reply:
I can probably have one script that targets both distros, i mainly wanted to have the dotnet install included in the scripts, these scripts have not changed in awhile and are based on what is in the Dockerfile for those environments so i rather not touch it right now.